CHANGELOG.rst

```diff
@@ -203,12 +210,6 @@ General
 Fixed
 ~~~~~
 
-:ref:`debops.docker_server` role
-''''''''''''''''''''''''''''''''
-
-- Do not add empty entries from `docker_server__listen` to daemon.json.
-  This causes the docker daemon to not parse the config and crash.
-
```

Comment:
I'm not sure why this is removed. You should probably rebase and force-push this PR against the `master` branch.